src/main/java/com/spring/springmvc/config/ProjectServiceInterceptorAppConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.spring.springmvc.config;
+
+import com.spring.springmvc.interceptors.LogInterceptor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+
+@Component
+public class ProjectServiceInterceptorAppConfig extends WebMvcConfigurerAdapter {
+    @Autowired
+    LogInterceptor logInterceptor;
```

Comment:
What about encapsulation? And why not use a constructor for DI.
Lombok could help you. Exmaple:  @Component@RequiredArgsConstructor(onConstructor = @__(@Autowired))
https://www.baeldung.com/spring-injection-lombok